python/tests/integration/test_async_dispatch_server_integration.py

```
@@ -57,39 +57,13 @@ def test_dispatch_probes_in_process():
 
 
 class _EchoHandler(socketserver.StreamRequestHandler):
+    # Lines received from the probes; asserted on the main thread because an
+    # AssertionError raised here is swallowed by socketserver.handle_error.
+    received: list[bytes] = []
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)